clarifai.go

```diff
@@ -37,7 +37,7 @@ func NewClarifaiClient() *ClarifaiClient {
 }
 
 // Check whether the client still has access rights to Clarifai
-func (client *ClarifaiClient) isAccess() bool {
+func isCLFAccessible(client *ClarifaiClient) bool {
```

Comment:
But you are right. Here it doesn't make sense to pass `clarifaiClient` as an argument. I should just use the global clarifaiClient inside the function so it takes no arguments